lib/vector-tap/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "vector-tap"
+version = "0.1.0"
+authors = ["Vector Contributors <vector@datadoghq.com>"]
+edition = "2021"
+publish = false
+license = "MPL-2.0"
+
+[dependencies]
+colored = { version = "2.1.0", default-features = false }
+serde_yaml = { version = "0.9.34", default-features = false }
```

Comment:
Out of scope for this PR. It would be nice to make some of these `[workspace.dependencies]`.